src/components/v2/appDetails/k8Resource/nodeType/nodeType.util.ts

```diff
@@ -1,19 +1,26 @@
-import { iNode } from '../../appDetails.type';
+import { Node, iNode } from '../../appDetails.type'
 
 export const getNodeStatus = (node: iNode) => {
     if (node.info && node.info.length > 0) {
         const statusReason = node.info.filter((_info) => {
-            return _info.name === 'Status Reason';
-        });
+            return _info.name === 'Status Reason'
+        })
         if (statusReason && statusReason.length > 0) {
-            return statusReason[0].value;
+            return statusReason[0].value
         }
     }
     if (node.status) {
-        return node.status;
+        return node.status
     }
     if (node.health?.status) {
-        return node.health?.status;
+        return node.health?.status
     }
-    return '';
-};
+    return ''
+}
+
+export const getNodeMessage = (node: Node) => {
+    if (node?.health?.message?.length > 0) {
```

Comment:
Can simply be written as `if (node?.health?.message){}`

Reply:
or single liner - `return node?.health?.message?.toLowerCase() ?? ''`

Reply:
Done